src/api/spec/controllers/public_controller_spec.rb

```diff
@@ -7,7 +7,7 @@
 
 RSpec.describe PublicController, vcr: true do
   let(:project) { create(:project, name: 'public_controller_project', title: 'The Public Controller Project') }
-  let(:package) { create(:package_with_file, name: 'public_controller_package', project: project) }
+  let!(:package) { create(:package_with_file, name: 'public_controller_package', project: project) }
```

Comment:
But the `before` block evaluated it

Reply:
I understand that. But why do you create the package for all test instead of the one in line 72-76?

Reply:
they create them anyway and so I had to type only one character :)